src/integration/data/bulk/bulkImport.test.ts

```diff
@@ -27,7 +27,6 @@ describe('bulk import', () => {
 
   afterAll(async () => {
     await retryDeletes(pinecone, indexName);
```

Comment:
This function deletes the index, so it was already deleted once the following line executed, thereby throwing the `404` error